app/controllers/organizations/adopter_fosterer/adopter_applications_controller.rb

```diff
@@ -50,7 +50,8 @@ def application_params
       :pet_id,
       :form_submission_id,
       :status,
-      :profile_show
+      :profile_show,
+      :person_id
```

Comment:
Why is this attribute added?

Reply:
Without, it throws 
```
 Unpermitted parameter: :person_id. Context: { controller: Organizations::AdopterFosterer::AdopterApplicationsController, action: create, request: #<ActionDispatch::Request:0x0000ffff601571c8>, params: {"authenticity_token"=>"[FILTERED]", "adopter_application"=>{"pet_id"=>"8", "person_id"=>"15"}, "commit"=>"Apply to Adopt", "controller"=>"organizations/adopter_fosterer/adopter_applications", "action"=>"create"} }
```
for me.

Reply:
Ah yes this might actually fix the production error I was seeing this morning 👯

Reply:
heck yeah